src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -102,5 +115,33 @@ protected override void QueueBlazorStart()
 					");
 			};
 		}
+
+		private void Webview_CoreWebView2Initialized(WebView2Control sender, UI.Xaml.Controls.CoreWebView2InitializedEventArgs args)
+		{
+			_webview.CoreWebView2.NavigationStarting += CoreWebView2_NavigationStarting;
+			_webview.CoreWebView2.NewWindowRequested += CoreWebView2_NewWindowRequested;
+		}
+
+		private void CoreWebView2_NavigationStarting(CoreWebView2 sender, CoreWebView2NavigationStartingEventArgs args)
+		{
+			if (Uri.TryCreate(args.Uri, UriKind.RelativeOrAbsolute, out var uri) &&
+				uri.Host != "0.0.0.0" && 
```

Comment:
See Android comment about creating a const for `0.0.0.0`: https://github.com/dotnet/maui/pull/4645/files#r807191768